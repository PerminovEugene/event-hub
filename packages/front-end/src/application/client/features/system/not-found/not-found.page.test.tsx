import * as React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import NotFound from './not-found.page';

test('NotFound page basic render', () => {
  const page = shallow(<NotFound />);
  expect(page.text()).toEqual('Not found page');
  expect(page).toMatchSnapshot();
});
