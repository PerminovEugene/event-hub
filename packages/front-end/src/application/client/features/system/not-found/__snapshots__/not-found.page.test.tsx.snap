// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound page basic render 1`] = `
<div>
  <styled.section>
    Not found page
  </styled.section>
</div>
`;
